library/x509_csr.c

```diff
@@ -100,6 +102,9 @@ static int x509_csr_parse_extensions(mbedtls_x509_csr *csr,
         extn_oid.p = *p;
         *p += extn_oid.len;
 
+        /* Get and ignore optional critical flag */
+        (void)mbedtls_asn1_get_bool(p, end_ext_data, &critical);
```

Comment:
```suggestion
        (void) mbedtls_asn1_get_bool(p, end_ext_data, &critical);
```
Code style checker failed as well.

Reply:
changed.